.travis.yml

```diff
@@ -58,19 +64,28 @@ matrix:
             - fonts-liberation
       env:
         - JOB=stability SCRIPT=tools/ci/ci_stability.sh PRODUCT=chrome:dev
-    - python: 2.7
+    - name: "tools unittests (Python 2)"
+      os: linux
+      python: "2.7"
       env: JOB=tools_unittest TOXENV=py27 HYPOTHESIS_PROFILE=ci SCRIPT=tools/ci/ci_tools_unittest.sh
-    - python: 3.6
+    - name: "tools unittests (Python 3)"
+      os: linux
+      python: "3.6"
       env: JOB=tools_unittest TOXENV=py36 HYPOTHESIS_PROFILE=ci SCRIPT=tools/ci/ci_tools_unittest.sh
-    - python: 2.7
+    - name: "resources unittests"
+      os: linux
+      python: "2.7"
       env: JOB=resources_unittest TOXENV=py27 SCRIPT=tools/ci/ci_resources_unittest.sh
-    - python: 2.7
+    - name: "flake8"
```

Comment:
I'm now pretty sure I've misnamed this one. This does involve `tools/wpt` and is probably what @jgraham referred to in https://github.com/web-platform-tests/wpt/pull/13300/files#r221878898. This does run flake8, but also other things it seems. @gsnedders, what should it be called? "tools/wpt/ pytest + flake8"?

Reply:
"wpt unittests", for consistency with all the other unittests jobs? (Or "wpi cli unittests", if that's unclear?)

Reply:
I'll go with tools/wpt/ unittests, if they are indeed accurate described as unit tests.